crates/rpc/src/routes/beacon.rs

```diff
@@ -28,6 +41,21 @@ pub fn get_beacon_routes(
         .and_then(move || get_genesis(network_spec.genesis.clone()))
         .with(log("genesis"));
 
+    let randao = {
+        beacon_base
+            .and(path("states"))
+            .and(param::<ID>())
+            .and(path("randao"))
+            .and(query::<RandaoQuery>())
+            .and(end())
+            .and(get())
+            .and(db_filter.clone())
+            .and_then(move |state_id: ID, query: RandaoQuery, db: ReamDB| {
+                get_randao_mix(state_id, query, db)
+            })
+            .with(log("randao"))
+    };
```

Comment:
```suggestion
    let randao =
        beacon_base
            .and(path("states"))
            .and(param::<ID>())
            .and(path("randao"))
            .and(query::<RandaoQuery>())
            .and(end())
            .and(get())
            .and(db_filter.clone())
            .and_then(move |state_id: ID, query: RandaoQuery, db: ReamDB| {
                get_randao_mix(state_id, query, db)
            })
            .with(log("randao"));
```

now that we don't do `let db = db.clone()` I don't think there is a reason to keep these brackets for scoping it